demo/tslint.json

```diff
@@ -52,7 +55,6 @@
     "no-switch-case-fall-through": true,
     "no-trailing-whitespace": true,
     "no-unused-expression": true,
-    "no-use-before-declare": true,
```

Comment:
I'd rather keep this one as is (even if it's deprecated), because for now you can still write es5 code pretty bad like
```js
console.log(a);
var a;
```
Also it's a good reminder for `eslint` migration to set `no-use-before-define` (https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-use-before-define.md)